packages/react/src/components/Slider/useSlider.ts

```diff
@@ -294,11 +351,12 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
     disabled,
   };
 
-  const thumbProps = {
-    ref: thumbRef,
-    className: classNames.thumb,
-    style: getPositionStyles(valuePercent),
-  };
+  const lowerValueLabelProps = ranged &&
```

Comment:
Should all of these props objects be typed? Otherwise the values aren't getting validated (and it makes the inferred types in the public API really messy).